api/types/helmchartargs.go

```diff
@@ -68,6 +68,10 @@ type HelmChart struct {
 	// Legal values: 'merge', 'override', 'replace'.
 	// Defaults to 'override'.
 	ValuesMerge string `json:"valuesMerge,omitempty" yaml:"valuesMerge,omitempty"`
+
+	// IncludeCRDs specifies if Helm should also generate CustomResourceDefinitions.
+	// Defaults to 'false'.
+	IncludeCRDs bool `json:"includeCRDs,omitempty" yaml:"includeCRDs,omitempty"`
```

Comment:
Would it be bad to make the default `true`?  Only require the user to do work if they wanted to suppress this?

For that matter, can we just drop this option and _always_ pass the `--include-crds`?

In what situation would that cause trouble?  Trying to understand the use case better.

Reply:
Hey hey. So I picked `false` as the default as to not break current behavior. I try not to submit patches that "break userland" and change the behavior that people may expect or unconsciously expect.

However! If you'd like me to make this the default, I'd be more than happy to. 

There's not really a situation that would cause trouble to my knowledge, other than if someone is currently using this expecting the CRDs to _not_ be there (albeit sort of silly.)

Reply:
I don't know enough about the implications; e.g. if helm barfs if the flag is used but there are no crds to include, it would be bad to always specify the flag.  So default off it is.